### DISCUSSION TITLE
What features do you want to see on JetNet?

### DISCUSSION THREAD

**Discussion by jaybdub** [NVIDIA-AI-IOT/jetnet#5] [Ideas]
Hi All,

We're interested to hear how you'd like to see JetNet grow.

Here are some thoughts we have, 

- Add more models (let us know which model you want!)
- Add more datasets (let us know which dataset you want to see!)
- Enable task specific datasets (ClassificationDataset)
- Enable model evaluation (ie: mAP, accuracy, etc.)
- Enable trainable models for using custom data

Please comment below with your new ideas, or copy and paste an idea already mentioned into your comment so we know it's something that interests you.

If you're interested in contributing, we also welcome contributions!  Even if PR's can't be accepted upstream, even just sharing your JetNet extension or project in a discussion will help others discover and learn from it.

Thanks!
John

**Comment by lynscott** [comment]
I don't see any comments here or many issues raised but I hope this is still an active project because I think it's great!

Having recently gone through the jetson hello world setup tutorial and then using tao to try and train my own models with custom data, it was pretty cumbersome. Especially the action recognition net.

This looks like it would've saved me a lot of time and energy!🥲

I'd love to see an action recognition model (for obvious reasons), and the ability to train on custom data.
